github/event_types.go

```diff
@@ -295,6 +295,18 @@ type PageBuildEvent struct {
 	Sender *User       `json:"sender,omitempty"`
 }
 
+// PingEvent is triggered when a WebHook is added to GitHub.
```

Comment:
Minor, but for consistency with what I see above and below, call it "Webhook", not "WebHook".